### ISSUE TITLE
Re-parent monitoring data under a server node to support multiple servers in a stripe

### ISSUE THREAD

**Issue by jd0-sag** [Terracotta-OSS/terracotta-apis#56]
The monitoring data currently passed to IMonitoringProducer is per-server.  This creates a limitation in terms of how data from passive servers can be accessed.

While there are some ideas for how to collect this data, on the active, that implementation is still a ways off.

We know, however, that this will result in the tree being re-parented under the server, in order to support multiple servers.  This gives each server its own monitoring namespace and also implicitly connects any information in the tree to the server where it is applicable.

In order to make the data shape consistent with that eventual re-parenting, we should re-parent even the single-server information we provide, in order to show that level of indirection.

**Comment by jd0-sag** [comment]
Note that this is somewhat low priority as it isn't absolutely required before passive servers need to be displayed.  Downstream teams are not yet blocked on this.
